packages/react-reconciler/src/__tests__/ReactIncrementalUpdates-test.js

```diff
@@ -189,58 +177,22 @@ describe('ReactIncrementalUpdates', () => {
     });
 
     // The sync updates should have flushed, but not the async ones.
-    if (
-      gate(
-        flags =>
-          !flags.forceConcurrentByDefaultForTesting &&
-          flags.enableUnifiedSyncLane,
-      )
-    ) {
-      assertLog(['d', 'e', 'f']);
-      expect(ReactNoop).toMatchRenderedOutput(<span prop="def" />);
-    } else {
-      // Update d was dropped and replaced by e.
-      assertLog(['e', 'f']);
-      expect(ReactNoop).toMatchRenderedOutput(<span prop="ef" />);
-    }
+    assertLog(['d', 'e', 'f']);
+    expect(ReactNoop).toMatchRenderedOutput(<span prop="def" />);
 
     // Now flush the remaining work. Even though e and f were already processed,
     // they should be processed again, to ensure that the terminal state
     // is deterministic.
-    if (
-      gate(
-        flags =>
-          !flags.forceConcurrentByDefaultForTesting &&
-          !flags.enableUnifiedSyncLane,
-      )
-    ) {
-      await waitForAll([
-        // Since 'g' is in a transition, we'll process 'd' separately first.
-        // That causes us to process 'd' with 'e' and 'f' rebased.
-        'd',
-        'e',
-        'f',
-        // Then we'll re-process everything for 'g'.
-        'a',
-        'b',
-        'c',
-        'd',
-        'e',
-        'f',
-        'g',
-      ]);
-    } else {
-      await waitForAll([
-        // Then we'll re-process everything for 'g'.
-        'a',
-        'b',
-        'c',
-        'd',
-        'e',
-        'f',
-        'g',
-      ]);
-    }
+    await waitForAll([
```

Comment:
This test has been updated to remove any test of the rebasing behavior. Is that because it no longer exists (unlikely) or because the test needs re-written to account for unifiedSyncLane?